package.json

```diff
@@ -34,7 +34,6 @@
     "crelt": "^1.0.5",
     "dapjs": "2.2.0",
     "dompurify": "^2.3.3",
-    "events": "^3.3.0",
```

Comment:
Requires `npm i` and another commit to udpate `package-lock.json`.

Reply:
Oops, thanks